### PR TITLE
Implement several object loading functions

### PR DESCRIPTION
**Pull request by AaronVanGeffen**

### CODE REVIEW

File: src/OpenLoco/src/Objects/StreetLightObject.cpp [hunk 1]
```diff
@@ -24,11 +28,21 @@ namespace OpenLoco
...
-        regs.ecx = enumValue(handle.type);
-        Interop::call(0x00477F19, regs);
+        auto remainingData = data.subspan(sizeof(StreetLightObject));
+
+        // Load object name string
+        auto strRes = ObjectManager::loadStringTable(remainingData, handle, 0);
+        name = strRes.str;
+        remainingData = remainingData.subspan(strRes.tableLength);
+
+        // TODO: verify what's happening at 0x00477F31
```
Comment (duncanspumpkin) [inline src/OpenLoco/src/Objects/StreetLightObject.cpp:38]: nothing. I suspect originally streetlights had a dependent object but then it was changed at a later date. TODO can be removed.

**Review by duncanspumpkin — APPROVED** [review]
Remove the todo and this is good to go.

File: src/OpenLoco/src/Objects/StreetLightObject.cpp [hunk 2]
```diff
@@ -24,11 +28,21 @@ namespace OpenLoco
...
+
```
Comment (AaronVanGeffen) [inline src/OpenLoco/src/Objects/StreetLightObject.cpp:39]:
```suggestion

```

File: src/OpenLoco/src/Objects/StreetLightObject.cpp (hunk 1 above)
Comment (AaronVanGeffen) [reply src/OpenLoco/src/Objects/StreetLightObject.cpp]: Alright, thanks for checking!

**Pull request merged**